experimental/generation/generator/templates/standard/datetimeEntity-date-time.lg

```diff
@@ -0,0 +1,4 @@
+[generator.lg](../generator.lg)
+
+# templates
+- ${entityTemplates()}
```

Comment:
I suspect we should name these files as dateTime instead of date-time because of the naming pattern.

Reply:
This is because the built-in jsom format is named date-time.